record-boot-success/com.endlessm.RecordBootSuccess.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Name=com.endlessm.RecordBootSuccess
+User=root
+SystemdService=record-boot-success.service
+Exec=/usr/lib/eos-boot-helper/record-boot-success/record-boot-success
```

Comment:
Are you sure that this is what `$libexecdir` (which is where this gets installed) expands to? This file should be templated, I think.

Reply:
Yes, I'm sure about the variable expansion, as I used it before for the factory reset scripts and fallback-fb-setup.

And when I wrote the first one of them I thought about having the variable in the service file and substituting at build time, but failed miserably to implement it. IIRC the variable was expanding to a different value in the service file that what it was in Makefile.am, but this was quite some time ago so I may be getting confused.

In any case, this package does not need to be super portable.

Reply:
It's not portability I'm worried about. I don't think it is a theoretical possibility that this path could become wrong.

Search https://manpages.debian.org/testing/debhelper/debhelper.7.en.html for "v12". Debian's policy about libexecdir has relatively recently changed – if we ever change `debian/compat` to `12` or greater for any other reason, `$(libexecdir)` in the `Makefile` will change to `/usr/libexec` and this will stop working, but the build will succeed. We'll only notice if we happen to read the journal.